ReText/__main__.py

```diff
@@ -92,6 +93,18 @@ def main():
 	fileNames = list(map(canonicalize, sys.argv[1:]))
 	previewMode = False
 	readStdIn = False
+    
+	if not QDBusConnection.sessionBus().registerService('org.ReText'):
```

Comment:
```suggestion
	if not QDBusConnection.sessionBus().registerService('me.mitya57.ReText'):
```

Let’s be consistent with the appdata and desktop files.